src/queries.js

```diff
@@ -1,109 +1,108 @@
-const {basicParser, contextRecordKey, deferred} = require('./utils');
+const {contextRecordKey, deferred} = require('./utils');
 
 function queriesStore(config, emitter, targetStore) {
-    const queries = {};
+  const queries = {};
 
-    async function getHandles(key, ids, params, context) {
-        if (!(key in queries)) queries[key] = [];
+  async function getHandles(key, ids, params, context) {
+    if (!(key in queries)) queries[key] = [];
 
-        let numCoalesced = 0;
-        let numCached = 0;
-        let numMisses = 0;
+    let numCoalesced = 0;
+    let numCached = 0;
+    let numMisses = 0;
 
-        const handles = [];
-        for (let i = 0; i < ids.length; i++) {
-            handles[i] = undefined;
+    const handles = [];
+    for (let i = 0; i < ids.length; i++) {
```

Comment:
[💨] It's a matter of style, but since we don't mutate the `i` index, we could use const.

Reply:
Because of the i++ in the loop, I don't think a const would work

Reply:
Totally right!